### PR TITLE
Apply monasca before ceilometer

### PR DESCRIPTION
**Pull request by gosipyan**

### CODE REVIEW

**Comment by gosipyan** [comment]
@toabctl its related to https://github.com/SUSE-Cloud/automation/pull/3284.

File: scripts/scenarios/cloud9/qa/no-ssl/qa-scenario-8a.yaml [hunk 1]
```diff
@@ -172,26 +172,10 @@ proposals:
...
-      - cluster:services
-      ceilometer-server:
-      - cluster:services
-      ceilometer-swift-proxy-middleware: []
-
 - barclamp: monasca
   attributes:
   deployment:
     elements:
-      monasca-master:
```
Comment (JanZerebecki) [inline scripts/scenarios/cloud9/qa/no-ssl/qa-scenario-8a.yaml]: Why is this being removed?

Comment (toabctl) [reply scripts/scenarios/cloud9/qa/no-ssl/qa-scenario-8a.yaml]: There is no longer a `monasca-master` role.

**Comment by toabctl** [comment]
> @toabctl its related to #3284.

then this should be described in the commit msg. anyway - assuming this was tested, I'm fine with merging it.

**Pull request merged**